src/main/resources/application.yaml

```diff
@@ -49,5 +56,13 @@ oidc:
 ccd:
   data-store:
     host: ${CCD_DATA_STORE_HOST:http://localhost:4452}
+    whitelisted-urls: /ccd/searcCases\?ctid.*
```

Comment:
ticket requested the following: 
![image](https://user-images.githubusercontent.com/9882391/87144131-300d7b80-c29f-11ea-8045-fec11d4e07f1.png)

Reply:
Finally added a test and set whitelisted-urls: ${MANAGE_CASE_DS_PROXY_URLS_WHITELIST:/ccd/searchCases.*,/ccd/internal/searchCases.*}

Reply:
Can we start using generic terminology like "AllowedList" ?